packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 ## Unreleased
 
 * Changed
-  * Changed ResultTimeHeaderCell to remove the `<time>` html tag for the date and time. This was causing issues and technically does not have full screen reader support.
+  * Changed ResultTimeHeaderCell to remove the `<time>` html tag for the date and time. This was causing issues and technically does not have full screen reader support.    
+  * Changed 3 translation keys (added 2 new keys and reused 1 existing)
```

Comment:
Consumers can't see the translation keys (outside of manually inspecting the code) so we don't have to be this specific.
```suggestion
  * Updated translations.
```